src/xtgeo/common/_angles.py

```diff
@@ -0,0 +1,57 @@
+from math import degrees, radians
+
+
+def _deg_angle2azimuth(deg: float) -> float:
+    """
+    Convert an angle in degrees, measured anti-clockwise from
+    the X-axis, to azimuth in degrees, measured clockwise from the Y-axis.
+
+    Parameters:
+    deg (float): The input angle in degrees, anti-clockwise from the X-axis.
+
+    Returns:
+    float: The converted azimuth in degrees, clockwise from the Y-axis.
+    """
```

Comment:
Although _files are not seen in official docs, I would prefer using google docstring style also here. See example here: https://github.com/equinor/xtgeo/blob/19a7988e3671970adf0c0a2e83b5b9efce94a9ac/src/xtgeo/surface/regular_surface.py#L1225

* Start with a one liner ("punch-line")
* Styles from here: https://google.github.io/styleguide/pyguide.html#doc-function-args

I.e. with type annotations, the type of each arg can be omitted.